lib/rules/shorthand-property-no-redundant-values/__tests__/index.js

```diff
@@ -115,6 +115,8 @@ testRule({
 			message: messages.rejected('1px 1px', '1px'),
 			line: 1,
 			column: 5,
+			endLine: 1,
+			endColumn: 21,
```

Comment:
[note] I don't think it's needed to change the arguments passed to `report()` since it should be helpful that the column range covers a declaration property and value.

https://github.com/stylelint/stylelint/blob/26f4c353079694584e469db7ca432bcc9a494fe3/lib/rules/shorthand-property-no-redundant-values/index.js#L170-L175

```css
a { margin: 1px 1px }
/*  ^^^^^^^^^^^^^^^
    Unexpected longhand value "1px 1px" instead of "1px"
*/
```